youtube_dl/extractor/bitchute.py

```diff
@@ -65,7 +65,7 @@ def _real_extract(self, url):
             webpage, default=None) or self._html_search_meta(
             'twitter:image:src', webpage, 'thumbnail')
         uploader = self._html_search_regex(
-            r'(?s)<p\b[^>]+\bclass=["\']video-author[^>]+>(.+?)</p>', webpage,
+            r'(?s)<div class=["\']channel-banner.*(?s)<p\b[^>]+\bclass=["\']name[^>]+>(.+?)</p>', webpage,
```

Comment:
Superfluous flags.

Reply:
I'm not sure I understand the comment, but please check ff24f33